README.md

```diff
@@ -37,6 +37,16 @@ So, make sure you have Hilt & kapt imported in your projects via below check-lis
    ```
 7. Now you're ready to use the Text to Speech function
 
+### How to publish a snapshot for local development & testing?
+
+1. Change `libVersion`, `versionCode` and publication `version` in https://github.com/elimu-ai/common-utils/blob/main/utils/build.gradle
```

Comment:
```suggestion
1. Change the `libVersion` and `versionCode` in https://github.com/elimu-ai/common-utils/blob/main/utils/build.gradle
```

Reply:
@tuancoltech I suggest removing "and publication version" since it looks like it's getting its value from the `libVersion`.

```gradle
release(MavenPublication) {
            groupId = group
            artifactId = artifact
            version = libVersion
            afterEvaluate {
                from(components["release"])
            }
        }
```